lib/questions.rb

```diff
@@ -208,6 +247,10 @@ def your_birthday_is_on_a_friday_in_the_year(birthday)
 # and 1 that is 4 letters long. Return it as a hash in the format
 # word_length => count, e.g. {2 => 1, 3 => 5, 4 => 1}
 def count_words_of_each_length_in_a_file(file_path)
+  words = File.foreach('data/lorem.txt').map { |line| line.split(' ') }.flatten.map{|word|word.chars.reject{|letter|letter=='.'|| letter==','}.join}
+  hash = {}
+  ((shortest_word_in_array(words).length)..(longest_word_in_array(words).length)).each{|numb| hash[numb] = words.select{|e| e.length==numb}.count }
```

Comment:
Line is too long. [147/80]<br>Surrounding space missing for operator '=='.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.